diff --git a/node_modules/react-native-modern-datepicker/src/datePicker/components/SelectMonth.js b/node_modules/react-native-modern-datepicker/src/datePicker/components/SelectMonth.js
index 6ccfa19..fbc4c85 100644
--- a/node_modules/react-native-modern-datepicker/src/datePicker/components/SelectMonth.js
+++ b/node_modules/react-native-modern-datepicker/src/datePicker/components/SelectMonth.js
@@ -76,13 +76,18 @@ const SelectMonth = () => {
   }, [prevDisable, nextDisable]);
 
   const onChangeYear = text => {
-    if (Number(utils.toEnglish(text))) {
+    if (text === '') {
+      setYear('');
+    }
+    else if (Number(utils.toEnglish(text))) {
       setYear(utils.toPersianNumber(text));
     }
   };
-
   const onSelectYear = number => {
     let y = Number(utils.toEnglish(year)) + number;
+    if (y === 0) {
+      y = new Date().getFullYear();
+    }
     if (y > selectorEndingYear) {
       y = selectorEndingYear;
     } else if (y < selectorStartingYear) {
diff --git a/node_modules/react-native-modern-datepicker/src/utils.js b/node_modules/react-native-modern-datepicker/src/utils.js
index c9f0cce..dc8aa7d 100644
--- a/node_modules/react-native-modern-datepicker/src/utils.js
+++ b/node_modules/react-native-modern-datepicker/src/utils.js
@@ -46,9 +46,9 @@ const gregorianConfigs = {
     'November',
     'December',
   ],
-  selectedFormat: 'YYYY/MM/DD',
-  dateFormat: 'YYYY/MM/DD',
-  monthYearFormat: 'YYYY MM',
+  selectedFormat: 'MM/DD/YYYY',
+  dateFormat: 'MM/DD/YYYY',
+  monthYearFormat: 'MM YYYY',
   timeFormat: 'HH:mm',
   hour: 'Hour',
   minute: 'Minute',
@@ -77,7 +77,7 @@ class utils {
 
   getFormated = (date, formatName = 'selectedFormat') => date.format(this.config[formatName]);
 
-  getFormatedDate = (date = new Date(), format = 'YYYY/MM/DD') => moment(date).format(format);
+  getFormatedDate = (date = new Date(), format = 'MM/DD/YYYY') => moment(date).format(format);
 
   getTime = (time) => this.getDate(time).format(this.config.timeFormat);
 
